web-ui/src/api/rolepermissions.js

```diff
@@ -0,0 +1,31 @@
+import { resolve } from "./api.js";
+
+const rolePermissionsListUrl = "/services/roles/role-permissions";
+
+export const getRolePermissionsList = async (cookie) => {
+  return resolve({
+    url: rolePermissionsListUrl,
+    responseType: "json",
+    headers: { "X-CSRF-Header": cookie },
+  });
+};
+
+export const postRolePermissionsList = async (roleData, cookie) => {
+  return resolve({
+    method: "post",
+    url: rolePermissionsListUrl,
+    responseType: "json",
+    data: roleData,
+    headers: { "X-CSRF-Header": cookie },
+  });
+};
+
+export const deleteRolePermissionsList = async (roleData, cookie) => {
+  return resolve({
+    method: "delete",
+    url: rolePermissionsListUrl,
+    responseType: "json",
+    data: roleData,
+    headers: { "X-CSRF-Header": cookie },
+  });
+};
```

Comment:
Should we omit the "List" from the names of these two functions? Posting and deleting role permissions takes in a single object with two IDs, so the naming is a bit confusing.

Reply:
Adjusted